PocketKit/Sources/PocketKit/Article/Cells/ArticleComponentTextCell.swift

```diff
@@ -25,19 +27,26 @@ extension ArticleComponentTextCell {
     func pocketTextView(_ textView: ArticleComponentTextView, shouldOpenURL url: URL) -> Bool {
         return delegate?.articleComponentTextCell(self, shouldOpenURL: url) ?? true
     }
+
+    func pocketTextView(_ textView: ArticleComponentTextView, contextMenuConfigurationForURL url: URL) -> UIContextMenuConfiguration? {
+        return delegate?.articleComponentTextCell(self, contextMenuConfigurationForURL: url)
+    }
 }
 
 // An object that conforms to this protocol is able to respond to (overridden)
 // events that occur within a PocketTextView.
 protocol ArticleComponentTextViewDelegate: AnyObject {
     func pocketTextViewDidSelectShareAction(_ textView: ArticleComponentTextView)
     func pocketTextView(_ textView: ArticleComponentTextView, shouldOpenURL url: URL) -> Bool
+    func pocketTextView(_ textView: ArticleComponentTextView, contextMenuConfigurationForURL url: URL) -> UIContextMenuConfiguration?
```

Comment:
Looks like we've had some naming drift when it comes to these methods. The class is `ArticleComponentTextView` but all the methods start with `pocketTextView(_:)`. Whatdya say we rename one or the other to keep things consistent? (I think renaming the methods makes the most sense here)